apps/src/blockly/addons/contextMenu.js

```diff
@@ -8,6 +8,7 @@ const HIDDEN = 'hidden';
 const MODERN = 'modern';
 const DARK = 'dark';
 const MUSICLAB_DARK = 'musiclabdark';
+const HIGH_CONTRAST = 'highContrast';
```

Comment:
What do you think of storing these constants in `blockly/constants.js`? You could do something like this...
```
export const ThemeOptions = {
  ENABLED: 'enabled',
  DISABLED: 'disabled',
  HIDDEN: 'hidden'
};
```
Could do something similar for the themes as well.

Reply:
Thanks Alice! I took your suggestion and made one tweak to names to better reflect what enabled/hidden/disabled are used for. I was also able to work the theme constants into the wrapper: https://github.com/code-dot-org/code-dot-org/pull/50297/commits/3bb1990b69a30a834863df33d9adbed781f47bd5

Reply:
Awesome! Thanks for the update!